.github/workflows/build-project.yaml

```diff
@@ -286,8 +286,13 @@ jobs:
 
           echo "cacheKey=${cache_key}" >> $GITHUB_OUTPUT
 
+      - name: Validate Flatpak manifest
+        run: |
+          : Validate Flatpak manifest
+          flatpak run --command=flatpak-builder-lint org.flatpak.Builder manifest ${{ github.workspace }}/build-aux/com.obsproject.Studio.json
```

Comment:
I don't recommend doing this, the JSON schema provided by the linter is not up to date against the one in the flatpak/flatpak-builder repo.

Reply:
We should just follow the most recent one

Reply:
You also need to install the Builder Flatpak, like I did in my WIP commit https://github.com/tytan652/obs-studio/commit/007bf8512d35820089f0ae583f9694eea2da804a